routes/api/users.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const router = express.Router();
+const auth = require("../../service/middleware/middleware");
+const {
+  signupUser,
+  loginOnUser,
+  logoutUser,
+  getCurrentUser,
+  updateSubscriptionUser,
+} = require("../controllers/manageUsers");
+
+router.post("/signup", signupUser);
+
+router.post("/login", loginOnUser);
+
+router.get("/logout", logoutUser);
```

Comment:
```suggestion
router.get("/logout", auth, logoutUser);
```
Tu też musisz to mieć jeśli żeby uzytkownika wylogować. Bo inaczej req.user jest pusty.